components/calendar/src/cal/julian.rs

```diff
@@ -500,4 +506,24 @@ mod test {
         Date::try_new_julian(-4, 2, 29).unwrap();
         Date::try_new_julian(2020, 2, 29).unwrap();
     }
+
+    #[test]
+    fn related_gregorian() {
+        // By 48901 the Julian calendar is a full year ahead
+        // of the Gregorian calendar
+        assert_eq!(
+            Date::try_new_julian(48900, 1, 1)
+                .unwrap()
+                .era_year()
+                .related_gregorian,
+            48900
+        );
+        assert_eq!(
+            Date::try_new_julian(48901, 1, 1)
+                .unwrap()
+                .era_year()
+                .related_gregorian,
+            48902
+        );
```

Comment:
Observation: Shouldn't this be the other way around? If the Julian year is shorter than the Gregorian year, there should be multiple Julian years that map to the same Gregorian year.

Reply:
The Julian calendar has more leap days, so the year is longer